antarest/study/storage/rawstudy/model/filesystem/root/output/simulation/mode/mcall/grid.py

```diff
@@ -5,10 +5,8 @@
 
 class OutputSimulationModeMcAllGrid(FolderNode):
     def build(self) -> TREE:
-        children: TREE = {
-            "areas": RawFileNode(self.context, self.config.next_file("areas.txt")),
-            "digest": RawFileNode(self.context, self.config.next_file("digest.txt")),
-            "links": RawFileNode(self.context, self.config.next_file("links.txt")),
-            "thermal": RawFileNode(self.context, self.config.next_file("thermal.txt")),
-        }
+        files = [d.stem for d in self.config.path.iterdir()]
```

Comment:
Je suggère qu'on utilise "folders" au lieu de "files". De même, on pourrait ajouter un `is_dir()` pour être certain du type.

Reply:
Here, we expect files and not folders.